content/src/service/deployments/DeploymentManager.ts

```diff
@@ -89,7 +88,7 @@ export class DeploymentManager {
     async getPointerChanges(deploymentPointerChangesRepo: DeploymentPointerChangesRepository, deploymentsRepo: DeploymentsRepository, filters?: PointerChangesFilters, offset?: number, limit?: number): Promise<PartialDeploymentPointerChanges> {
         const curatedOffset = (offset && offset >= 0) ? offset : 0
         const curatedLimit = (limit && limit > 0 && limit <= DeploymentManager.MAX_HISTORY_LIMIT) ? limit : DeploymentManager.MAX_HISTORY_LIMIT
-        const deploymentsWithExtra = await deploymentsRepo.getHistoricalDeploymentsByLocalTimestamp(curatedOffset, curatedLimit + 1, filters)
+        const deploymentsWithExtra = await deploymentsRepo.getHistoricalDeployments(curatedOffset, curatedLimit + 1, filters)
```

Comment:
Could you specify that you need the local timestamp? I don't want to rely on the default